src/shapes/Image.ts

```diff
@@ -782,13 +780,14 @@ export class Image<
    * @param {AbortSignal} [options.signal] handle aborting, see https://developer.mozilla.org/en-US/docs/Web/API/AbortController/signal
    * @returns {Promise<Image>}
    */
-  static fromObject(
-    { filters: f, resizeFilter: rf, src, crossOrigin, ...object }: any,
+  static fromObject<T extends TProps<SerializedImageProps>>(
+    { filters: f, resizeFilter: rf, src, crossOrigin, ...object }: T,
     options: { signal: AbortSignal }
-  ): Promise<Image> {
+  ) {
     return Promise.all([
       loadImage(src, { ...options, crossOrigin }),
       f && enlivenObjects(f, options),
+      // TODO: redundant - handled by enlivenObjectEnlivables
       rf && enlivenObjects([rf], options),
```

Comment:
we have been visiting this line a few times

Reply:
i prefer it clearly states what is going on here.
If we need it to enlive filters, i want to enlive also the resizeFilter manually. is more clear

Reply:
That can be said regarding the rest of the enlivenables...
I don't mind but it reminds me of that open PR #8507 that touches that logic